packages/design-system/src/utils/prepareCookiesCount.ts

```diff
@@ -61,11 +61,11 @@ const prepareCookiesCount = (cookies: { [key: string]: CookieData } | null) => {
   ).length;
 
   cookiesCount.blockedCookies.total = cookieList.filter(
-    (cookie) => cookie.isBlocked
+    (cookie) => cookie.blockedReasons && cookie.blockedReasons?.length > 0
```

Comment:
Doesn't it make `isBlocked` value redundant if its value can be derived from blockedReason everywhere else too?

Reply:
@ayushnirwal added the reasoning in the comment above.